test/issue-generator.js

```diff
@@ -56,8 +56,10 @@ export default function createTestIssue(key, type, color, parentKey, parentType,
           }
         }
       });
+    case 'Deployment Task':
     case 'Initiative':
     case 'Bug':
+    case 'Unknown':
     case 'Maintenance Task':
     // This is old??
     case 'MT':
```

Comment:
Perhaps this is a bit of a gripe, but rather than all these cases should this just be a default case?